woocommerce/woocommerce
# Fix typo in in-app marketplace sections conditional

### All Submissions:






### Changes proposed in this Pull Request:

On the **WooCommerce > Marketplace > Shipping** page, the conditional statement that checks the current section has a typo and there's no specific class for the gateways section.

Also, the list of countries where WooCommerce Payments is available has been updated.

This PR addresses those issues.

Closes 11246-gh-Automattic/woocommerce.com

### How to test the changes in this Pull Request:

- Check out this branch on your local environment
- Update the store country / state to any US state from the [settings](https://woocommerce.test/wp-admin/admin.php?page=wc-settings)
- Visit the in-app marketplace [page](https://woocommerce.test/wp-admin/admin.php?page=wc-addons&section=shipping-methods)
- Go to the "Shipping" section from the dropdown select
- Confirm you see the featured "WooCommerce Shipping" large card (see screenshot below)
- Please test around this issue

### Screenshots

![](https://d.pr/i/NEZ9yR+)

### Other information:




### Changelog entry

> fix in-app marketplace conditional statements

### FOR PR REVIEWER ONLY:
